src/components/header.jsx

```diff
@@ -4,7 +4,7 @@ import { VscGithubAlt } from "react-icons/vsc";
 let HeaderComponent = () => {
   return (
     <>
-      <nav className="mt-4 h-auto mx-2 rounded-md bg-[#31ce9f]">
+      <nav className="h-auto rounded-b-md bg-[#31ce9f]">

```

Comment:
@cool-pants I think the border should be evenly rounded.

Reply:
Having the border evenly rounded leaves gaps at the top giving an unfinished look.